### DISCUSSION TITLE
LINUX VS WINDOWS

### DISCUSSION THREAD

**Discussion by Yezid27** [hackatomik/SISCOx256#9] [Preguntas Técnicas | Lab] [answered]
Si Linux es mejor 
Por qué a día de hoy Windows es mas popular y la que más vemos con regularidad

**Comment by AlejandraMichellx** [comment, marked as answer]
A mi pensar, Windows es más popular porque casi todas las computadoras vienen con ello ya instalado. También, las empresas y escuelas usan Windows, así que mucha gente está acostumbrada a él desde pequeños. 

He llegado a usar una computadora con Linux al igual que con Windows, y en mi opinión personal prefiero Windows ya que estoy más familiarizado con el, el cual puede sea el caso de muchos.

**Reply by tapilew** [reply]
Sip, la costumbre.

Pero en el mundo de los servidores, Linux tiene casi el dominio absoluto. Por eso muchas herramientas para programar se crean con Linux (y MacOS que también se basa en Unix) en mente. Es mucho más cómodo programar con Linux que Windows. Si hablamos de gaming... se invierten los roles haha.

Si uno prefiere Windows y quiere programar ahí no hay ningún problema. Para eso se creó WSL, una terminal y sistema Linux disponible para ser ejecutado en una máquina Windows.
